### PR TITLE
added russian, extracted some resources

### PR DESCRIPTION
**Pull request by EntyR**

### CODE REVIEW

File: app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<resources>
+    <string name="app_name">RoomDb</string>
+    <string name="add_new_random_number">Добавить новое случайное число</string>
+    <string name="back">Назад</string>
+    <string name="number_is_even">Число четное</string>
+    <string name="number_not_even">число нечетное</string>
+    <string name="no_numbers">Нет чисел</string>
+    <string name="number">Число %1$d</string>
+    <string name="numberdescription">Число %1$d</string>
+    <string name="return_back_button">Кнопка для возвращения обратно</string>
```
Comment (AlexeyKolosov) [inline app/src/main/res/values-ru/strings.xml]: please use `translatable false` for strings, which are not displayed

File: app/src/main/java/com/harman/roomdbapp/app/ui/fragments/RandomNumberAddItem.kt [hunk 2]
```diff
@@ -49,8 +49,9 @@ class RandomNumberAddItem : Fragment() {
         })
 
         binding.etEnterNumber.setOnKeyListener { view, i, event ->
+
             if ((event.action == KeyEvent.ACTION_DOWN) &&
-                (i == KeyEvent.KEYCODE_ENTER)
+                ((i == KeyEvent.KEYCODE_ENTER))
```
Comment (AlexeyKolosov) [inline app/src/main/java/com/harman/roomdbapp/app/ui/fragments/RandomNumberAddItem.kt]: Could you please explain, what it is for? (I mean double parentheses)

Comment (EntyR) [reply app/src/main/java/com/harman/roomdbapp/app/ui/fragments/RandomNumberAddItem.kt]: Overlooked it. was fixing enter button on tabletes.

File: app/src/main/res/values-ru/strings.xml (hunk 1 above)
Comment (EntyR) [reply app/src/main/res/values-ru/strings.xml]: done

File: app/src/main/java/com/harman/roomdbapp/app/ui/fragments/RandomNumberAddItem.kt (hunk 2 above)
Comment (EntyR) [reply app/src/main/java/com/harman/roomdbapp/app/ui/fragments/RandomNumberAddItem.kt]: done

**Review by AlexeyKolosov — APPROVED** [review]

**Pull request merged**